packages/mdc-toolbar/mdc-toolbar.scss

```diff
@@ -78,13 +86,43 @@ $mdc-toolbar-mobile-breakpoint: 599px;
   &__title {
     @include mdc-typography(title);
 
+    align-self: center;
     margin: 0;
+    margin-left: 16px;
+    padding: $mdc-toobar-element-vertical-padding 0;
     line-height: 1.5rem;
     text-overflow: ellipsis;
     white-space: nowrap;
     overflow: hidden;
     z-index: 1;
   }
+
+  &__icon {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    padding: $mdc-toobar-element-vertical-padding 8px;
+    border: none;
+
+    @include mdc-theme-prop(color, text-primary-on-primary);
+  }
+
+  &__icon--menu {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    padding: $mdc-toobar-element-vertical-padding;
+    border: none;
+
+    @include mdc-theme-prop(color, text-primary-on-primary);
+  }
+
+  &__icon__button {
```

Comment:
It doesn't work because anchor tags and buttons are nested inside of `mdc-toolbar__icon`, et al

I think what might work though is just setting `mdc-toolbar__icon` styles directly to the anchors, spans, and buttons